src/fping.c

```diff
@@ -2937,12 +2937,12 @@ void usage(int is_error)
     fprintf(out, "   -6, --ipv6         only ping IPv6 addresses\n");
     fprintf(out, "   -b, --size=BYTES   amount of ping data to send, in bytes (default: %d)\n", DEFAULT_PING_DATA_SIZE);
     fprintf(out, "   -B, --backoff=N    set exponential backoff factor to N (default: 1.5)\n");
-    fprintf(out, "   -c, --count=N      count mode: send N pings to each target\n");
```

Comment:
It's a bit unfortunte that -c also changes the output format but it is definitely also changing the way probing is done. I am thinking that maybe we should either remove the option sections, or possibly introduce a new one, like "Operation mode" or something like that, where only -c, -C, and -l are documented. Otherwise, I would probably leave -c where it is now.

Reply:
I think that it would be good to have both `-c` and `-C` in the same section, because they both affect probing and output similarly.

For now, I suggest I'll redo the pull request without the change regarding where `-c` is shown in the help output.